app/controllers/application.js

```diff
@@ -114,6 +114,13 @@ export default class ApplicationController extends ParachuteController {
 
   loadStateTask = task(function* () {
     yield timeout(500);
+
+    const map = this.get('map');
+
+    if(map) {
```

Comment:
looks like this solved it!

so the center was being set before the map rendered on the page?

Reply:
we weren't setting it after the map was loaded. Here, it is just making sure the map is loaded before this event handler references it.